Model/Sales/Package.php

```diff
@@ -59,6 +59,11 @@ class Package extends Data implements PackageInterface
      */
     private $all_products_fit = true;
 
+    /**
+     * @var bool
+     */
+    private $all_products_fit_in_digital_stamp = true;
```

Comment:
```suggestion
    private $fit_in_digital_stamp = true;
```

Reply:
Ben ik niet met je eens, in de functie setFitInDigitalStampFromQuoteProducts zit er namelijk een loop die kijkt of alle producten in een digitale postzegel kunnen. Het zijn dus meerdere producten waar hij op controleert.

Reply:
Done!